### DISCUSSION TITLE
Assign authors mentioned in release notes

### DISCUSSION THREAD

**Discussion by wyozi** [renovatebot/renovate#22203] [Suggest an Idea]
Especially in micro-frontend configurations it sometimes makes the most sense to assign the person who built the original change in the original repository as the reviewer for the version bump PR as well. There are tools to automatically tag the authors in release notes (e.g. github-release-notes), but these get converted to `togithub.com` links and mentions are also weaker than assigning as a reviewer. Therefore, it would be nice to automatically assign the users that were mentioned in release notes as reviewers.

**What would you like Renovate to be able to do?**

Assign all github usernames as reviewers that have been mentioned in release notes

**Did you already have any implementation ideas?**

Could be `{ "reviewersFromReleaseNotes": true }`

**Comment by rarkins** [comment]
Self hosted or GitHub.com? Such a capability would be painful if misused or misunderstood on the latter

**Comment by wyozi** [comment]
@rarkins I was thinking for Github.com. Would the misuse potential be in having non-organizational members be assigned as reviewers or the amount of notification spam?

What about limiting it to private repositories?

**Comment by rarkins** [comment]
Notification spam is a big risk. I think it would be safe if we made it opt-in and private repos only. The worst case then is that you spam your colleagues.
